README.md

```diff
@@ -53,6 +55,7 @@ $ docker-compose up
 ```
 *Both storefront and dashboard are quite big frontend projects and it might take up to few minutes for them to compile depending on your CPU. If nothing shows up on port 3000 or 9000 wait until `Compiled successfully` shows in the console output.*
 
+*If you experience a timeout error, increase timeout variables of Docker to 5 minutes.*
```

Comment:
`timeout variables` - googling this doesn't give me immediate answer on which variable(s) it is, maybe we can paste it here? Otherwise seems a bit cryptic to me.